cmd/grumble/message.go

```diff
@@ -1328,7 +1328,7 @@ func (server *Server) handleUserStatsMessage(client *Client, msg *Message) {
 	stats.Session = proto.Uint32(target.Session())
 
 	if details {
-		if tlsconn := target.conn.(*tls.Conn); tlsconn != nil {
+		if tlsconn, ok := target.conn.(*tls.Conn); ok {
```

Comment:
I'd add a comment about not supporting client certificates for WebSocket connections here, too.